voluncheer/jobboard/urls.py

```diff
@@ -1,10 +1,13 @@
 from django.urls import path
 
-from jobboard import views
+from jobboard.views.jobboard import jobboard, select
+from jobboard.views.postajob import post_a_job
+
 
-app_name = "jobboard"
 urlpatterns = [
-    path("", views.jobboard, name="jobboard"),
+    path("", jobboard, name="jobboard"),
     # Jobboard
-    path("select", views.select, name="select"),
+    path("select", select, name="select"),
+    # Post a Job
+    path("post_a_job", post_a_job, name="post_a_job"),
```

Comment:
On the back-end, I'm fine with job as a technical/short-hand term, but anything user-facing should definitely use opportunity. So perhaps the URL should be "opportunities/" for the job board and "opportunities/post" for the submission form. 

As for what we refer to it in the code, the only reasons I see to keep it as "job" is that it's slightly shorter to write job and it matches better with the job board (which we are are more likely say than "opportunity board"). But, in the interest of longevity, perhaps it is better to call everything opportunity to so that someone from the outside wouldn't be confused by jobs vs. opportunities.

Long story short, I agree we should change everything to opportunity/opportunities.